companion/manifest.json

```
@@ -0,0 +1,26 @@
+{
+  "id": "novastar-splicer",
+  "name": "novastar-splicer",
+  "shortname": "novastar-splicer",
+  "description": "Companion module for the splicer of Novastar",
+  "version": "1.0.0",
+  "license": "MIT",
+  "repository": "git+git+https://github.com/bitfocus/companion-module-novastar-h-series.git",
+  "bugs": "git+https://github.com/bitfocus/companion-module-novastar-h-series.git/issues",
```

Which will mean this will need updating too:
```suggestion
  "repository": "git+https://github.com/bitfocus/companion-module-novastar-splicer.git",
  "bugs": "https://github.com/bitfocus/companion-module-novastar-splicer/issues",
```

Either way the current bugs link in this PR isn't valid/doesn't link to the issues. I'm pretty certain the repo link is wrong too, maybe too much search/replace?